test/instruments_previous_test.go

```diff
@@ -6,96 +6,133 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
+	"github.com/checkout/checkout-sdk-go/common"
 	"github.com/checkout/checkout-sdk-go/errors"
 	"github.com/checkout/checkout-sdk-go/instruments"
 	"github.com/checkout/checkout-sdk-go/instruments/abc"
 	"github.com/checkout/checkout-sdk-go/tokens"
 )
 
 func TestCreateAndGetInstrumentPrevious(t *testing.T) {
```

Comment:
This should be different tests. You should test the Create method first and validate the create response, and then you should do another function to test the Get method with a previously created Instrument.

Reply:
Like there should be two different functions:
- TestCreateInstrument
- TestGetInstrument

Reply:
But for get an instrument, we need to create one, and this creation method is called from more than one test, this is the reason.